mlrun/builder.py

```diff
@@ -402,7 +402,14 @@ def build_image(
             source = parsed_url.path
             to_mount = True
             source_dir_to_mount, source_to_copy = path.split(source)
-        else:
```

Comment:
so in case of `else` here, we need to raise an exception because source is a path without a schema and not abs? meaning, source is a local? is that ok (local runs?)